packages/gcf-utils/package.json

```diff
@@ -28,9 +28,12 @@
     "@types/ioredis": "^4.14.8",
     "@types/lru-cache": "^5.1.0",
     "@octokit/plugin-enterprise-compatibility": "1.2.5",
+    "@types/pino": "^6.3.0",
```

Comment:
I'm surprised so many of our types are `dependencies`, vs., `devDependencies`. @orthros remind me if there was a reason for this?

Reply:
I think @JustinBeckwith just did that because users of gcf-utils generally needed those types (#647)

Reply:
Yup.

Reply:
sounds good.